src/test/java/com/rarchives/ripme/tst/ripper/rippers/EromeRipperTest.java

```diff
@@ -51,7 +51,7 @@ public void testRip() throws IOException, URISyntaxException {
     @Test
     @Tag("slow")
     public void testVideoAlbumRip() throws IOException, URISyntaxException {
-        URL url = new URI("https://www.erome.com/a/P0x5Ambn").toURL();
+        URL url = new URI("https://www.erome.com/a/jEUFu6pi").toURL();
```

Comment:
The old URL seems to be valid as well, but the new one is an album containing 2 videos which seems like a better test (and more SFW which is good).